notes/git-guidelines-commit-and-push.md

```diff
@@ -0,0 +1,74 @@
+## GUIDELINES TO GIT COMMIT AND PUSH
+
+Committing and pushing changes in Git involve several best practices to ensure a clean and organized version history. Here are some guidelines and best practices:
+
+### COMMITTING
+
+1. always check if you are in the right branch
+
+```bash
+    git branch
+```
+
+2. Add files with `git add .` (for all files) or `git add <file_name>` (for specific file)
+3. Check if the files you want are supposed to be staged with
+
+```bash
+git status
+```
+
+2. Review changes before committing with
```

Comment:
I guess the terminal is not the best way to check diff, confusing, what about git graph or another visual tool?